packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method-form.jsx

```diff
@@ -114,34 +122,38 @@ const AddPaymentMethodForm = ({ should_show_separated_footer = false }) => {
                                         />
                                     )}
                                 </Field>
-                                {my_profile_store.selected_payment_method_fields &&
-                                    my_profile_store.selected_payment_method_fields.map((payment_method_field, key) => {
-                                        return (
-                                            <Field
-                                                name={payment_method_field[0]}
-                                                id={payment_method_field[0]}
-                                                key={key}
-                                            >
-                                                {({ field }) => (
-                                                    <Input
-                                                        {...field}
-                                                        data-lpignore='true'
-                                                        error={errors[payment_method_field[0]]}
-                                                        type={
-                                                            payment_method_field[0] === 'instructions'
-                                                                ? 'textarea'
-                                                                : payment_method_field[1].type
-                                                        }
-                                                        label={payment_method_field[1].display_name}
-                                                        className='add-payment-method-form__payment-method-field'
-                                                        onChange={handleChange}
+                                <div>
```

Comment:
Why do we need to wrap this in an empty div?

Reply:
Forgot to remove it. Had added to test a CSS issue